packages/device_info/README.md

```diff
@@ -0,0 +1,26 @@
+# device_info_tizen
+
+The Tizen implementation of [`device_info`](https://github.com/flutter/plugins/tree/master/packages/device_info).
+
+## Usage
+
+```yaml
+dependencies:
+  device_info_tizen: ^1.0.0
+```
+
+You can import `device_info` in your Dart code:
```

Comment:
`device_info` → `device_info_tizen`

Reply:
Done.

Reply:
Typo deviceinfo -> device_info

If you are to upload multiple plugins the README file can be updated later at once. By the way, is this plugin not supported on TV?

Reply:
Technically... It was not tested at TV. Currently I haven't got a commercial binary TV device, and the tv emulator doesn't work. I don't know if this is my problem or the embedder problem.